numpy/distutils/system_info.py

```diff
@@ -1698,14 +1705,15 @@ def calc_info(self):
             lib = self.has_cblas(info)
             if lib is not None:
                 info['language'] = 'c'
-                info['libraries'] = [lib]
+                info['libraries'] = lib
                 info['define_macros'] = [('HAVE_CBLAS', None)]
         self.set_info(**info)
 
     def has_cblas(self, info):
+        """ changed in 1.17 to return a list instead of a str """
```

Comment:
Can you move this change to another PR? The changes to this function seem unrelated to the renaming you do above.

Reply:
Ok, I'll do that, I'll need to extract the entire CBLAS change then.

Reply:
See #13161